src/swap/utils/sendSwapTransactions.test.ts

```diff
@@ -61,21 +59,20 @@ describe('sendSwapTransactions', () => {
       config,
       sendTransactionAsync,
       sendCallsAsync,
-      setCallsId,
       updateLifecycleStatus,
       walletCapabilities: { [Capabilities.AtomicBatch]: { supported: true } },
       transactions,
     });
     expect(sendCallsAsync).toHaveBeenCalledTimes(1);
     expect(sendCallsAsync).toHaveBeenCalledWith({ calls: transactions });
-    expect(setCallsId).toHaveBeenCalledWith('callsId');
     expect(updateLifecycleStatus).toHaveBeenCalledTimes(2);
     expect(updateLifecycleStatus).toHaveBeenNthCalledWith(1, {
       statusName: 'transactionPending',
     });
     expect(updateLifecycleStatus).toHaveBeenNthCalledWith(2, {
       statusName: 'transactionApproved',
       statusData: {
+        callsId: 'callsId',
         transactionType: 'Batched',
```

Comment:
@alessey if we want to use `lifecycle`, let's emit the `callsId` as part of `transactionApproved`

Reply:
I think this is fine since with a `callsId` as a developer, you can still use this to query the calls status yourself using the RPC `wallet_getCallsReceipt` (granted you know which wallet server to use)

Reply:
I would imagine we can include a field here based on the Wagmi connector, e.g. `walletServer` or `walletConnector` if the developer wants to use these wallet-specific fields